### PR TITLE
committing initial code

### PR DESCRIPTION
**Pull request by samrdd**
Sending for PR

### CODE REVIEW

File: Assignment6/Assignment6.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+
```
Comment (panchalashish4) [inline Assignment6/Assignment6.sh:4]: You can add a sleep command to wait for the MariaDB container to start.

File: Assignment6/Assignment6.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    echo "Database exist"
+    mariadb -u root -ppassword -h mariadb1 baseball < Assignment6.sql
+    mariadb -u root -ppassword -h mariadb1 baseball < Assignment6.sql > mystuff/file_output.csv
+
+else
+    echo "Database does not exist, Creating database"
+    mariadb -u root -ppassword -h mariadb1 -e "CREATE DATABASE baseball;"
+    mariadb -u root -ppassword -h mariadb1 baseball < baseball.sql
+    mariadb -u root -ppassword -h mariadb1 baseball < Assignment6.sql
+    mariadb -u root -ppassword -h mariadb1 baseball < Assignment6.sql > mystuff/file_output.csv
```
Comment (panchalashish4) [inline Assignment6/Assignment6.sh]: Nice method to get the output file.

**Review by panchalashish4 — CHANGES_REQUESTED** [review]
1. You can put docker-compose and Dockerfile in the main project folder outside the Assignment6 folder so that we don't have to do **cd Assignment6** to go inside the folder to run and while doing COPY command you can mention the folder for files.
2. In the Assignment6.sh file you can put a sleep command to wait for MariaDB to start.
3. Pre-commit failed.
4. Other than that, it works for me after starting the MariaDB container first and then running the baseball container. I'm able to get the output file.

**Comment by samrdd** [comment]
Made changes

**Pull request closed without merging**